services/rollout-service/pkg/cmd/server.go

```diff
@@ -31,6 +31,7 @@ import (
 	pkgmetrics "github.com/freiheit-com/kuberpult/pkg/metrics"
 	"github.com/freiheit-com/kuberpult/pkg/setup"
 	"github.com/freiheit-com/kuberpult/pkg/tracing"
+	"github.com/freiheit-com/kuberpult/services/rollout-service/pkg/datadogdora"
```

Comment:
please provide a description for this PR, e.g. mention 
https://docs.datadoghq.com/dora_metrics/ (if that's the right link)